pandas/core/index.py

```diff
@@ -1,3 +1,8 @@
 # flake8: noqa
-from pandas.core.indexes.api import *
+from pandas.core.indexes.api import (
+    CategoricalIndex, DatetimeIndex, Float64Index, Index, Int64Index,
```

Comment:
is there a reason . you changed this here?

Reply:
If this line will remain the same, the following errors will occur:
```
mypy.exe pandas\core\tools\datetimes.py
pandas\core\tools\datetimes.py:101: error: Module 'pandas' has no attribute 'DatetimeIndex'
pandas\core\tools\datetimes.py:101: error: Module 'pandas' has no attribute 'Index'
```